go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/spiffe/spire
 
 go 1.19
 
+replace github.com/spiffe/spire-api-sdk => github.com/guilhermocc/spire-api-sdk v0.0.0-20230313160835-e14db6e1ffb4
```

Comment:
https://github.com/spiffe/spire-api-sdk/pull/38 need to be merged before this PR

Reply:
I'm updating this branch right away